Launcher/config.json

```diff
@@ -65,6 +65,9 @@
   // save list of transactions to the specified csv file
   "transaction-log": "",
 
+  // Prefix for Windows reserved filenames
+  "reserved-words-prefix": "",
```

Comment:
Should add the default value of `@` 👍, having it empty will cause it not to use the default here `Config.Get("reserved-words-prefix", "@")`, which will only be used if not present, worth a test 💯